mscore/preferences.cpp

```diff
@@ -313,7 +313,7 @@ void Preferences::write()
       s.setValue("myImagesPath", myImagesPath);
       s.setValue("myTemplatesPath", myTemplatesPath);
       s.setValue("myPluginsPath", myPluginsPath);
-      s.setValue("sfPath",  sfPath);
+      s.setValue("mySoundfontsPath", mySoundfontsPath);
```

Comment:
I want to make sure you are aware that I have changed the MuseScore_.ini string name from "sfPath" to "mySoundfontsPath".  This will mean users will have to re-add their own soundfont folders after this commit.  I think mySoundfontsPath is more descriptive than "sfPath" and I think it is important to change the name here in the ini to reflect that this setting shouldn't include the mscoreGlobalShare/sounds.